package.json

```diff
@@ -127,5 +129,8 @@
     "presets": [
       "react-app"
     ]
+  },
+  "@std/esm": {
+    "esm": "js"
```

Comment:
I could not work out how to get Babel to *not* transpile modules. So rather than have `@std/esm` handle everything, it will only handle the convert script. Setting this to `js` means it will parse everything as an 'es6' module.